arroyo-sql-testing/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 mod full_query_tests;
+mod smoke_tests;
```

Comment:
I think it's worth putting this (and probably full_query_tests as well) behind a flag. On my machine, this branch takes 60 to check from clean, compared to 45 for master and 40 seconds without full_query_tests.